lib/internal/main/worker_thread.js

```diff
@@ -135,6 +139,27 @@ port.on('message', (message) => {
     if (!hasStdin)
       process.stdin.push(null);
 
+    if (directStdout) {
+      process.stdout._writev = function _writev(chunks, cb) {
+        const { chunk, encoding } = ArrayPrototypePop(chunks);
+        write(1, chunk, { __proto__: null }, encoding, (err) => {
```

Comment:
This won't work on Windows with non-ASCII text, unfortunately

Reply:
I would love to land this PR. Is this non-ASCII thing an issue that needs resolution beyond just documenting as a known limitation?

Assuming that it is, could we use whatever solution `console.log` itself uses for outputting to stdout for Windows?